### PR TITLE
Rebrand security products

### PR DESCRIPTION
**Pull request by dcurwin**

### CODE REVIEW

File: api-reference/beta/api/alert-list.md [hunk 1]
```diff
@@ -52,12 +52,12 @@ The following table lists the `$filter` keywords by each vendor name.
 
 | Vendor name      |$filter keyword|
 |:----------|:----------|
-| Azure Advanced Threat Protection | Azure Advanced Threat Protection | 
+| Microsoft Defender for Identity | Azure Advanced Threat Protection |
```
Comment (isvargasmsft) [inline api-reference/beta/api/alert-list.md:55]: 1. Could you please confirm if these other products are being rebranded as well? If so, shouldn't we update the second column of this table too?

File: api-reference/beta/api/alert-list.md [hunk 2]
```diff
@@ -52,12 +52,12 @@ The following table lists the `$filter` keywords by each vendor name.
 
 | Vendor name      |$filter keyword|
 |:----------|:----------|
-| Azure Advanced Threat Protection | Azure Advanced Threat Protection | 
+| Microsoft Defender for Identity | Azure Advanced Threat Protection | 
 | Azure Security Center | ASC |
-| Microsoft Cloud App Security | MCAS |
+| Microsoft Defender for Cloud Apps | MCAS |
```
Comment (isvargasmsft) [inline api-reference/beta/api/alert-list.md:57]: Please update the second column to display MDCA

File: api-reference/beta/api/alert-list.md (hunk 1 above)
Comment (dcurwin) [reply api-reference/beta/api/alert-list.md:55]: I am not responsible for the API, so I can't speak to that. Please take this offline with your engineers.

File: api-reference/beta/api/alert-list.md (hunk 2 above)
Comment (dcurwin) [reply api-reference/beta/api/alert-list.md:57]: MDCA is not an approved acronym for Microsoft Defender for Cloud Apps. I am not responsible for the API, so I don't know what should appear in that column. Please take this offline with your engineers.

File: api-reference/beta/api/alert-list.md (hunk 1 above)
Comment (dcurwin) [reply api-reference/beta/api/alert-list.md:55]: No, I made the changes as they should be made.

**Comment by isvargasmsft** [comment]
Waiting for technical validation from the product PMs.

**Review by dcurwin — COMMENTED** [review]
approved

**Comment by isvargasmsft** [comment]
@Erikre some Intune docs need to be updated to reflect the recent rebranding of Microsoft products. Could you please take a look and include them in your tools. I understand intune PRs are handled separately.

**Pull request merged**